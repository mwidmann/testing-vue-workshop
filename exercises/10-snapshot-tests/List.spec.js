import List from '../2-components/List.vue'
import { shallowMount } from '@vue/test-utils';

test('renders correctly when there are no items', () => {
  const wrapper = shallowMount(List)
  expect(wrapper.element).toMatchSnapshot()
})

test('renders correctly when there are three items', () => {
  const wrapper = shallowMount(List, {
    propsData: {
      items: [ 'item1', 'item2', 'item3' ]
    }
  })
  expect(wrapper.element).toMatchSnapshot()
})
