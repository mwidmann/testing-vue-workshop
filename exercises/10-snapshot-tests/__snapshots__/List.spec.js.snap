// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly when there are no items 1`] = `
<span>
  No items
</span>
`;

exports[`renders correctly when there are three items 1`] = `
<ul>
  <li>
    
    item1
  
  </li>
  <li>
    
    item2
  
  </li>
  <li>
    
    item3
  
  </li>
</ul>
`;
